packages/dom/src/clone-dom.js

```diff
@@ -84,20 +84,20 @@ export function cloneNodeAndShadow(ctx) {
 /**
  * Use `getInnerHTML()` to serialize shadow dom as <template> tags. `innerHTML` and `outerHTML` don't do this. Buzzword: "declarative shadow dom"
  */
-export function getOuterHTML(docElement) {
-  // All major browsers in latest versions supports getHTML API to get serialized DOM
-  // https://developer.mozilla.org/en-US/docs/Web/API/Element/getHTML
-  // old firefox doesn't serialize shadow DOM, we're awaiting API's by firefox to become ready and are not polyfilling it.
-  // new firefox from 128 onwards serializes it using getHTML
-  /* istanbul ignore if: Only triggered in firefox <= 128 and tests runs on latest */
-  if (!docElement.getHTML) { return docElement.outerHTML; }
```

Comment:
-> Need to remove this, as this was blocking snapshot CLI flow, as our discovery is Chrome 123, and `getHTML` is only supported from Chrome 125 onwards